lib/kubeclient/common.rb

```diff
@@ -202,6 +205,14 @@ def resource_name(entity_type)
         entity_type.pluralize.downcase
       end
 
+      def proxy_url_service(namespace, service, protocol)
```

Comment:
@jimmidyson nope. The usage of that function is proxy_url_service('default', 'image-scanner', '5001-tcp'), which builds the uri http://<host>:<port>/api/v1/proxy/namespaces/default/services/image-scanner:5001-tcp

I'm not sure how websockets are relevant here.